dev/TreeView/TreeViewListAutomationPeer.cpp

```diff
@@ -44,4 +45,26 @@ winrt::IInspectable TreeViewListAutomationPeer::GetPatternCore(winrt::PatternInt
 winrt::AutomationControlType TreeViewListAutomationPeer::GetAutomationControlTypeCore()
 {
     return winrt::AutomationControlType::Tree;
-}
\ No newline at end of file
+}
+
+// ISelectionProvider
+bool TreeViewListAutomationPeer::CanSelectMultiple()
+{
+    return IsMultiselect()? true: __super::CanSelectMultiple();
+}
+
+bool TreeViewListAutomationPeer::IsSelectionRequried()
+{
+    return IsMultiselect()? false : __super::CanSelectMultiple();
+}
+
+winrt::com_array<winrt::Windows::UI::Xaml::Automation::Provider::IRawElementProviderSimple> TreeViewListAutomationPeer::GetSelection()
+{
+    // The selected items might be collapsed, virtualized, so getting an accurate list of selected items is not possible.
+    return winrt::com_array<winrt::Windows::UI::Xaml::Automation::Provider::IRawElementProviderSimple>();
```

Comment:
Take advantage of C++'s default construction syntax. Since the return type is specified, the context can be used to infer the type so simplify this line to:

```suggestion
    return {};
```